fastlane/Fastfile

```diff
@@ -596,6 +596,15 @@ platform :ios do
       )
     end
 
+    # We're about to use `add_development_certificates_to_provisioning_profiles` and `add_all_devices_to_provisioning_profiles`.
+    # These actions use Developer Portal APIs that don't yet support authentication via API key (-.-').
+    # Let's preemptively ask for and set the email here to avoid being asked twice for it if not set.
+    asc_user_email_key = 'FASTLANE_USER'
+    unless ENV.key?(asc_user_email_key)
+      UI.important("#{asc_user_email_key} value not found in the environment.")
```

Comment:
```suggestion
      UI.important("#{asc_user_email_key} environment variable not set. Set and `export` it in your `~/.zshrc` to avoid being asked about it next time.")
```

I find the "in the environment" wording a bit unclear without additional context. Is that "environment" in the sense of "the current repository" or something? So I personally prefer using the term `environment variable` to make things clearer.